uni/lib/view/bug_report/widgets/form.dart

```diff
@@ -57,12 +60,17 @@ class BugReportFormState extends State<BugReportForm> {
   bool _isConsentGiven = false;
 
   void loadBugClassList() {
-    bugList = [];
+    final locale = Intl.getCurrentLocale();
```

Comment:
I would say avoid working with locale strings manually at all costs. Provide a `AppLocale` enum model and make the provider (perhaps in a static method) do this logic.